docs/team/junrong98.md

```diff
@@ -8,34 +8,49 @@ title: Jun Rong's Project Portfolio Page
 NUS Classes is an app that enables the professors to better manage contacts from large numbers of students and staff, and allow the professor to document the task that he/she needs to do.
 Given below are my contributions to the project.
 
-** Will update in future**
-
-* **New Feature**: to be added soon
-    * What it does: to be added soon
-    * Justification: to be added soon
-    * Highlights: to be added soon
-    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
-
+* **New Feature**: View Task
+    * What it does: Allows user to view who were assigned to each task.
+    * Justification: User have to know who were assigned to each task, so users can easily grab the required information.
+    * Highlights: Understanding how `ObservableList` works and the effect it has on the GUI.
+    * Credits: *{code ideas from AB3 contacts feature}*
+
+* **New Feature**: Create recurring tasks
+    * What it does: Allows user to create a task repeatedly with a fixed interval between them.
+    * Justification: Convenience to user as it allows user to create tasks that occur regularly with a single command.
+
+* **New Feature**: List tasks
+    * What it does: List all tasks.
+    * Justification: Allows users to see all the tasks created by them, especially after they manipulated to task list.
+  
```

Comment:
Not sure if `Create recurring tasks` can be considered a "New Feature", feels like its more of an upgrade to an existing feature.

Reply:
IMO, i think is consider new feature as this is work on Add task feature(which is a new feature)